cranelift/codegen/src/isa/zkasm/abi.rs

```diff
@@ -473,25 +468,16 @@ impl ABIMachineSpec for ZkAsmMachineDeps {
 
     fn gen_clobber_save(
         _call_conv: isa::CallConv,
-        flags: &settings::Flags,
+        _flags: &settings::Flags,
         frame_layout: &FrameLayout,
     ) -> SmallVec<[Self::I; 16]> {
         let mut insts = SmallVec::new();
-        // Adjust the stack pointer downward for clobbers and the function fixed
+        // Adjust the stack pointer upward for clobbers and the function fixed
         // frame (spillslots and storage slots).
         let stack_size = frame_layout.fixed_frame_storage_size + frame_layout.clobber_size;
-        // Each stack slot is 64 bit and can fit a u8 value.
+        // The stack (and memory in general) in zkASM is addressed in slots, rather than bytes.
```

Comment:
Am I right that I can think here ``slot`` ~= ``machine word``

Reply:
Roughly, yeah. Between the two the differences are mostly pedantic and implied. For example “machine word” does not necessarily imply aligned accesses – “machine word” would better describe the size of the memory access and it could very well be unaligned too. “slot” does a great job of implying that the memory itself is segmented into chunks (implying that you cannot really load half a word from one slot and half a word from the next one in a single operation.)